spec/swagger-developer.yaml

```diff
@@ -1685,7 +1690,7 @@ host: localhost:8443
 info:
   description: Cloudentity Authorization Control Plane Public API
   title: Cloudentity Authorization Control Plane API
-  version: 1.13.1
+  version: latest
```

Comment:
Oops - need to set a correct version.

Reply:
I notice that `version: latest` comes from ACP - should i fix these to a particular release, or is 'latest' OK?

Reply:
latest is ok